src/models/Dexes.ts

```diff
@@ -0,0 +1,31 @@
+import {index, modelOptions, prop, Severity} from '@typegoose/typegoose';
+
+@index({chainId: 1})
+@index({dexProtocol: 1})
+@index({token0: 1})
+@index({token1: 1})
+@index({pool: 1})
+@index({chainId: 1, dexProtocol: 1, token0: 1, token1: 1})
```

Comment:
we should also add unique indexes: 
- chainId, fee, token0, token1
- chainId, pool